src/main/scala/io/sdkman/changelogs/java/AzulZuluMigrations.scala

```diff
@@ -257,5 +257,97 @@ class AzulZuluMigrations {
         removeVersion(candidate = "java", version = "11.0.6-zulu", platform)
     )
   }
+  @ChangeSet(order = "009", id = "009-add_zulu_7_0_262", author = "jorsol")
+  def migrate009(implicit db: MongoDatabase) = {
+    List(
+      Version(
+        "java",
+        "7.0.262-zulu",
+        "https://cdn.azul.com/zulu/bin/zulu7.38.0.11-ca-jdk7.0.262-linux_x64.tar.gz",
+        Linux64,
+        Some(Zulu)
+      ),
+      Version(
+        "java",
+        "7.0.262-zulu",
+        "https://cdn.azul.com/zulu/bin/zulu7.38.0.11-ca-jdk7.0.262-win_x64.zip",
+        Windows,
+        Some(Zulu)
+      ),
+      Version(
+        "java",
+        "7.0.262-zulu",
+        "https://cdn.azul.com/zulu/bin/zulu7.38.0.11-ca-jdk7.0.262-macosx_x64.zip",
```

Comment:
We only support zips for Windows at this time. Could you change this to be a tarball please?